sherpa/astro/io/__init__.py

```diff
@@ -56,8 +74,8 @@
 config.read(get_config())
 
 # What should we use for the default package?
-io_opt = config.get('options', 'io_pkg', fallback='pyfits')
-io_opt = str(io_opt).strip().lower()
+io_opt = config.get('options', 'io_pkg', fallback='dummy')
+io_opt = [str(o).strip().lower() + '_backend' for o in io_opt.split()]
```

Comment:
```suggestion
io_opt = [o.strip().lower() + '_backend' for o in io_opt.split()]
```

I'm pretty sure we don't need the str here. I don't know why it was in the old version (probably defensive programmning).